apps/dashboard/src/components/editor/document-settings.js

```diff
@@ -3,13 +3,60 @@
  */
 // eslint-disable-next-line import/named
 import { DocumentSection } from 'isolated-block-editor';
-// import { GlobalStylesUI } from '@wordpress/edit-site/src/components/global-styles';
-
-const DocumentSettings = () => (
-	<DocumentSection>
-		Great things will come here
-		{ /*<GlobalStylesUI />*/ }
-	</DocumentSection>
-);
+import { get } from 'lodash';
+import { __ } from '@wordpress/i18n';
+// eslint-disable-next-line
+import { __experimentalGetSettings, format } from '@wordpress/date';
+import { ExternalLink, PanelBody, PanelRow } from '@wordpress/components';
+
+const DocumentSettings = ( { project } ) => {
+	const DATETIME_FORMAT = 'F j, Y H:i';
```

Comment:
NIt: this constant and `formatDate` could be defined outside of `DocumentSettings` since they don't actually rely on anything from the component in their implementation. Let's just put them above it.